Automattic/wp-calypso
# A4A > Migrations: Implement UI for Tag sites for commission

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1443

## Proposed Changes

This PR implements the Tag sites for commission functionality for Migrations. Please note this is a partial implementation. Clicking on the `Add X Sites` button doesn't trigger any action.

Note: The table checkbox has some existing issues, which are not part of this PR. They will be fixed separately. 

It also creates components like `A4AModal` & `A4ATablePlaceholder`.

<img width="816" alt="Screenshot 2024-11-12 at 3 06 06 PM" src="https://github.com/user-attachments/assets/12108979-b4de-4f67-a13c-5152bd8a8daf">

## Why are these changes being made?

* To allow users to tag a site as migrated. 

## Testing Instructions

* Open the A4A live link
* Go to Overview > Click the `Add Sites` button on the top right > Select the `Via WordPress.com` option > Verify the loading indicator looks good since we replaced it with the new `A4ATablePlaceholder` component.
* Go to Migrations > Overview
* Click the `Tag sites for commission` button > Verify all the sites displayed on the dashboard are shown. Also, verify you can select multiple sites. Note: Clicking on the `Add X Sites` button doesn't trigger any action, this will be implemented in another PR.

## Pre-merge Checklist
